qiskit/pulse/builder.py

```diff
@@ -451,28 +450,37 @@ def call_schedule(self, schedule: Schedule):
 
     def call_subroutine(self,
                         subroutine: Union[circuit.QuantumCircuit, Schedule],
-                        value_dict: Optional[Dict[ParameterExpression, ParameterValueType]] = None
-                        ):
-        """Call a schedule defined outside of the current scope.
+                        name: Optional[str] = None,
+                        **kw_params):
```

Comment:
```suggestion
                        **kw_params,
                        value_dict=None):
```
For consistency, we should also support `value_dict` for cases many parameters may share a name.